apps/web/src/services/administration/store/role-page-store.ts

```diff
@@ -25,11 +26,12 @@ export const useRolePageStore = defineStore('role-page', {
         selectedRoles: (state) => state.selectedIndices.map((d) => state.roles[d]) || [],
     },
     actions: {
-        async listRoles(apiQuery: Query) {
+        async listRoles(params: RoleListParameters) {
+            const { query } = params;
             this.loading = true;
             try {
-                const res = await SpaceConnector.clientV2.identity.role.list({
-                    query: apiQuery,
+                const res = await SpaceConnector.clientV2.identity.role.list<RoleListParameters>({
```

Comment:
```suggestion
                const res = await SpaceConnector.clientV2.identity.role.list<RoleListParameters, ListResponse(RoleModel)>({
```
We can also specify the type of response.